hera_sim/noise.py

```diff
@@ -67,22 +69,29 @@ def white_noise(size=1):
     )
 
 
+# XXX reverse fqs and lsts in this function?
 def resample_Tsky(fqs, lsts, Tsky_mdl=None, Tsky=180.0, mfreq=0.18, index=-2.5):
     """
-    Evaluate sky temperature as a function of LST and frequency.
+    Re-sample a model of the sky temperature at particular freqs and lsts.
```

Comment:
it is not clear to me what "resample" means here -- to me it indicates statistical sampling, but this function seems to only evaluate the sky temperature?

Reply:
Tsky_mdl is an interpolator, so you are resampling one set of data onto a new fqs/lsts grid